techniques/failures/F94.html

```diff
@@ -27,13 +27,13 @@ <h2>Examples</h2>
 			<section class="example">
 				<h3>Failure example 1</h3>
 				<p>The following CSS and HTML snippet uses VW units to size the text.</p>
-<pre xml:space="preserve"><code class="language-css">/* CSS */
+<pre><code>/* CSS */
 .callout {
   font-size:1vw;
 }</code>
 
-<code class="language-html">&lt;p class="callout"&gt;Text that scales by viewport units&lt;/p&gt;</code></pre>
-				<p class="working-example">Example <a href="../../working-examples/failure-viewport-units/">page with an example of text sized in <code class="language-css">vh</code> units</a>.</p>
+<code>&lt;p class="callout"&gt;Text that scales by viewport units&lt;/p&gt;</code></pre>
```

Comment:
```suggestion
<pre><code>&lt;p class="callout"&gt;Text that scales by viewport units&lt;/p&gt;</code></pre>
```

here it DOES make sense to split these into two code blocks, but might as well do separate `pre` elements as well (so highlighting can pick up the difference between css and html highlighting